cmd/multifile/manifest_test.go

```diff
@@ -49,58 +49,49 @@ func TestCheckSeenDests(t *testing.T) {
 	}
 
 	// a different destination is fine
-	err := checkSeenDests(seenDests, "/tmp/file2.txt", "https://example.com/file2.txt")
+	err := checkSeenDestinations(seenDests, "/tmp/file2.txt", "https://example.com/file2.txt")
 	assert.NoError(t, err)
 
 	// the same destination with a different URL is not fine
-	err = checkSeenDests(seenDests, "/tmp/file1.txt", "https://example.com/file2.txt")
+	err = checkSeenDestinations(seenDests, "/tmp/file1.txt", "https://example.com/file2.txt")
 	assert.Error(t, err)
 
 	// the same destination with the same URL is also not fine
-	err = checkSeenDests(seenDests, "/tmp/file1.txt", "https://example.com/file1.txt")
+	err = checkSeenDestinations(seenDests, "/tmp/file1.txt", "https://example.com/file1.txt")
 	assert.Error(t, err)
 }
 
 func TestAddEntry(t *testing.T) {
```

Comment:
What is this test for? What is it testing? Can it have a better name?

Also, if it's testing AddEntry, it should be in the package where AddEntry is defined